online_meetings/canvas_zoom_meetings.py

```diff
@@ -160,16 +160,30 @@ def get_zoom_course(self, course: canvasapi.course.Course) -> None:
                 self.get_zoom_details(posturl, formdata, course.id)
         return None
 
-    def zoom_course_report(self, canvas_account: int = 1, enrollment_term_id: int = 0,
-                           published: bool = True, add_course_ids: list = None) -> None:
+    def zoom_course_report(
+        self,
+        canvas_account: int = 1,
+        enrollment_term_ids: Sequence[int] = [],
```

Comment:
You should avoid using a mutable default value as an argument. This should follow this pattern. https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
This was mentioned in the Codacy review as an issue as well.

Reply:
Thank you for flagging this and sharing the link, I'm happy to have read and learned about the anti-pattern. I think I've resolved it here, but there are two other occurrences of this that I introduced to the codebase. I'll open a separate PR to address those.

Reply:
Ah, that makes sense.  Codacy (PyLint) referred to it as a problem with `%s`.  I was distracted by that weirdness and I didn't think of the real potential problem.  I think I would automatically code it with the sentinel value and I've never thought of what could be the problem with specifying the empty list as default argument.

Reply:
Yeah, I think it's a "pretty cool feature" until you run into it in actual code accidentally. ;) Shouldn't use empty lists or dicts as default args.